pkg/grub/rootfs.cfg

```diff
@@ -245,7 +255,7 @@ function set_arm64_baremetal {
       # this makes grepping for XR320 our best option
       if regexp -- "XR320" "$smb_product"; then
          set_global dom0_console "console=ttyAMA0,115200n8"
-         set_global dom0_platform_tweaks "pcie_aspm=off pci=pcie_bus_perf crashkernel=auto"
+         set_global dom0_platform_tweaks "pcie_aspm=off pci=pcie_bus_perf"
```

Comment:
Should this also be removed from pkg/eve/installer/ipxe.efi.cfg?

Reply:
Missed that, will remove.